packages/create-catalyst/jest.config.cjs

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  transformIgnorePatterns: [],
```

Comment:
Unit tests throw `unexpected token` errors for `import` keywords in CLI dependency source files if this line is not here 🤔 By default, [Jest does not transform files in `node_modules`](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases). 

This feels like a bandaid, open to suggestions for how to fix this differently

Reply:
This does strike me as awkward but I don't have a better solution.